pom.xml

```diff
@@ -80,7 +81,7 @@
         <dependency>
             <groupId>com.carrotsearch.randomizedtesting</groupId>
             <artifactId>randomizedtesting-runner</artifactId>
-            <version>2.1.13</version>
+            <version>2.1.14</version>
```

Comment:
should we make this a constant / property in the pom file since it's used twice?